image/generic/sle16/root/etc/migration-config.yml

```diff
@@ -1 +1,3 @@
 soft_reboot: false
+network:
+  wicked2nm-continue-migration: true
```

Comment:
In my opinion this should be temporary and I added it for easier testing. I would keep the default to false, however until at least https://github.com/openSUSE/wicked2nm/issues/40 is fixed it is pretty much necessary to set this to true as otherwise even a simple dhcp connection will issue a warning.